provisioning/ubuntu-provision.sh

```diff
@@ -154,6 +154,9 @@ function install_asdf() {
   echo ". ${asdf_install_dir}/asdf.sh" >> "${profile_script}"
   chown -R "${username}:${groupname}" "${userhome}"
   rm -f "${archive}"
+
+  ## append to the system wide path variable, need to be seconded for docker in packer sources.pkr.hcl
+  sed -e "/^PATH/s/\"$/:\${asdf_install_dir}\/shims:\${asdf_install_dir}\/bin\"/g" -i /etc/environment
```

Comment:
There is a `\` in front of the first `$` here while there isn't any line 212, is it expected?

Reply:
it's a double quote interpolated string while the other one is without interpolation, if I surround everything by simple quotes, it will disable interpolation. The `\` is to escape the used double quote `"`